zanata-war/src/main/java/org/zanata/rest/editor/MediaTypes.java

```diff
@@ -32,6 +32,11 @@ public String toString() {
     public static final String APPLICATION_ZANATA_LOCALES_JSON =
             APPLICATION_ZANATA_LOCALES + JSON;
 
+    public static final String APPLICATION_ZANATA_SUGGESTIONS =
+            APPLICATION_VND_ZANATA + ".suggestions";
+    public static final String APPLICATION_ZANATA_SUGGESTIONS_JSON =
+            APPLICATION_ZANATA_SUGGESTIONS + JSON;
+
```

Comment:
We needn't tackle cleaning up the adjacent media types right now, but I think SuggestionsResource should produce APPLICATION_ZANATA_SUGGESTIONS_JSON and (on second thought, David) generic APPLICATION_JSON.  We don't need APPLICATION_ZANATA_SUGGESTIONS as such.  The client should request 
APPLICATION_ZANATA_SUGGESTIONS_JSON.

Note that the current code doesn't actually appear to use APPLICATION_ZANATA_SUGGESTIONS_JSON anywhere.